src/bmeg/models/vertex_models.py

```diff
@@ -63,9 +69,9 @@ def __post_init__(self):
     def make_gid(cls, genome, chromosome, start, end, reference_bases,
                  alternate_bases):
         # TODO -  figure out better hashing strategy
-        vid = "%s:%s:%d:%d:%s:%s" % (genome, chromosome,
-                                     start, end, reference_bases,
-                                     alternate_bases)
+        vid = "{}:{}:{}:{}:{}:{}".format(genome, chromosome, start, end,
```

Comment:
You're losing a little fidelity here, since `%d` is forcing an integer and will fail if it's not an int (e.g. is "start" is a string). If you really want to use `.format` you need `{0:d}` but I think %d is simpler.

Reply:
changed it to format() so I could chain the encode()

Reply:
changed to %